iati/core/tests/test_resources.py

```diff
@@ -1,6 +1,7 @@
 """A module containing tests for the library implementation of accessing resources."""
 from lxml import etree
 import pytest
+from six import string_types
```

Comment:
This issue has now been resolved in later commits and the PR is now ready for further review.